setup.py

```diff
@@ -181,7 +182,8 @@ def write_version(filename=os.path.join(*['airflow',
     'nose-ignore-docstring==0.2',
     'nose-parameterized',
     'nose-timer',
-    'rednose'
+    'rednose',
+    'redis'
```

Comment:
I could remove it from `devel` and leave it to be installed as an extra feature, like others, like `emr` or `postgres`, etc. Doing `pip install "airflow[redis]"` but that would need also an update on the documentation (`docs/installation.rst`). 
What do you think @artwr  ?